lib/Driver/DarwinToolChains.cpp

```diff
@@ -431,12 +396,13 @@ toolchains::Darwin::constructInvocation(const LinkJobAction &job,
 
   // Link compatibility libraries, if we're deploying back to OSes that
   // have an older Swift runtime.
-  Optional<std::pair<unsigned, unsigned>> runtimeCompatibilityVersion;
+  Optional<llvm::VersionTuple> runtimeCompatibilityVersion;
   
   if (context.Args.hasArg(options::OPT_runtime_compatibility_version)) {
-    auto value = context.Args.getLastArgValue(options::OPT_runtime_compatibility_version);
+    auto value = context.Args.getLastArgValue(
+                                    options::OPT_runtime_compatibility_version);
```

Comment:
Nitpick (here and elsewhere): double-indent for wrapped lines rather than reverse-indenting until it fits.

Reply:
`clang-format` should DTRT